### PR TITLE
[Mobile] Fix page templates button alignment and strings

### LINKED ISSUE #2287: [GlobalStep] [Android] "Get in Touch" button is misaligned as well as "Exclamation point symbol" is missing from "Let's build something together" text in default page templates for "About" and "Services" .
**Environment:**
Android: Samsung Galaxy s9 (v10), Samsung Galaxy S7 (v8), Samsung Galaxy Tab A (v9).
Build WPAndroid [version]: v14.9

**Steps To Reproduce:**
 1. Install and launch the application on both the platforms.
 2. Tap on "Log In".
 3. Log in to the app using valid credentials.
 4. Tap on Site Pages option.
 5. Tap on Add page "+" icon.
 6. Select About/Services page template and tap on apply button.
 7. Tap on Publish button.
 8. Pop-up appears tap on "Publish now".
 9. Now tap on three dots symbol and tap on view option.
 10. Observe that "Get in Touch" button is not centrally aligned and" Exclamation point symbol" also missing with this "Let's build something together "text.
 
 Note: Compared the template with this link.
 Link: https://github.com/wordpress-mobile/gutenberg-mobile/issues/2168#issuecomment-616857093

**Actual Result:**
 "Get in Touch" button is misaligned as well as "Exclamation point symbol" is missing from "Let's build something together" text in default page templates for "About" and "Services" .

 **Expected Result:**
 Application should display "Get in Touch" button centrally aligned as well as "Exclamation point symbol" should be present for "Let's build something together" text.

**Attachment:**
[#2287.zip](https://github.com/wordpress-mobile/gutenberg-mobile/files/4662985/2287.zip)

### PR DESCRIPTION
**Pull request by mkevins**
Fixes: https://github.com/wordpress-mobile/gutenberg-mobile/issues/2287 https://github.com/wordpress-mobile/gutenberg-mobile/issues/2295 https://github.com/wordpress-mobile/gutenberg-mobile/issues/2267

### Related PRs

`gutenberg-mobile`: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2314

## Description
This PR fixes some strings in the mobile pages templates to be consistent, and also uses center alignment for the buttons.

## How has this been tested?
In a new page, apply each of the templates and verify that the buttons are center aligned, and that the following strings are consistent "Let's build something together!" (should have an exclamation mark at the end), and "Get in Touch" (consistently title case for each instance).

## Types of changes
"Bug" fix (non-breaking change which fixes an issue)

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

File: packages/block-editor/src/components/page-template-picker/templates/about.native.js [hunk 1]
```diff
@@ -109,12 +109,13 @@ const About = {
...
 				align: 'center',
 				// translators: sample content for "About" page template
-				content: __( 'Let’s build something together' ),
+				content: __( 'Let’s build something together!' ),
 				level: 2,
 			},
 		},
 		{
 			name: 'core/buttons',
+			align: 'center',
```
Comment (geriux) [inline packages/block-editor/src/components/page-template-picker/templates/about.native.js:118]:
```suggestion
			"attributes": {
				"align": "center"
			},
```

File: packages/block-editor/src/components/page-template-picker/templates/portfolio.native.js [hunk 2]
```diff
@@ -184,19 +184,20 @@ const Portfolio = {
...
 				align: 'center',
 				// translators: sample content for "Portfolio" page template
-				content: __( "Let's build something together." ),
+				content: __( "Let's build something together!" ),
 				level: 2,
 			},
 		},
 		{
 			name: 'core/buttons',
+			align: 'center',
```
Comment (geriux) [inline packages/block-editor/src/components/page-template-picker/templates/portfolio.native.js:193]:
```suggestion
			"attributes": {
				"align": "center"
			},
```

File: packages/block-editor/src/components/page-template-picker/templates/services.native.js [hunk 3]
```diff
@@ -148,12 +148,13 @@ const Services = {
...
 				align: 'center',
 				// translators: sample content for "Services" page template
-				content: __( 'Let’s Build Something Together' ),
+				content: __( 'Let’s build something together!' ),
 				level: 2,
 			},
 		},
 		{
 			name: 'core/buttons',
+			align: 'center',
```
Comment (geriux) [inline packages/block-editor/src/components/page-template-picker/templates/services.native.js:157]:
```suggestion
			"attributes": {
				"align": "center"
			},
```

File: packages/block-editor/src/components/page-template-picker/templates/team.native.js [hunk 4]
```diff
@@ -226,13 +226,14 @@ const Team = {
 		},
 		{
 			name: 'core/buttons',
+			align: 'center',
```
Comment (geriux) [inline packages/block-editor/src/components/page-template-picker/templates/team.native.js:229]:
```suggestion
			"attributes": {
				"align": "center"
			},
```

**Review by geriux — APPROVED** [review]
LGTM! Tested it on both iOS and Android ✅

**Comment by mkevins** [comment]
Thanks for reviewing and testing Gerardo!

**Pull request merged**